drivers/overlay/overlay.go

```diff
@@ -90,6 +94,14 @@ func Init(dc driverapi.DriverCallback, config map[string]interface{}) error {
 		}
 	}
 
+	if port, ok := config[VxlanPortLabel]; ok {
```

Comment:
The vxlan port config almost use for whole cluster, not only for one overlay network. So, According to the @mavenugo comment, we should make the node's advertise its vxlan port, and use the destination node's vxlan port to build the fdb forward table.